api-specification/COUNTER_API.json

```diff
@@ -17490,4 +17490,4 @@
         "securitySchemes": {}
     },
     "x-internal": false
-}
```

Comment:
Could you please remove the newline at the end of the file? We keep the file in sync with stoplight.io which seems to ignore whitespace at the end of the file.